automated/linux/armnn/armnn-unit-tests.yaml

```diff
@@ -0,0 +1,23 @@
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: armnn unit tests
+    description: "Carry out armnn unit tests."
+    maintainer:
+        - theodore.grey@linaro.org
+    os:
+        - debian
+    scope:
+        - functional
+    devices:
+        - synquacer
+        - dragonboard-845c
+        - hi960-hikey
+        - stm32mp1
+params:
+    LINK_SNAPSHOT_64BIT: 'https://snapshots.linaro.org/componenets/armnn/latest/armnn.tar.xz'
+    LINK_SNAPSHOT_32BIT: 'https://snapshots.linaro.org/componenets/armnn-32bit/latest/armnn-32.tar.xz'
```

Comment:
maybe just have one variable like ARMNN_TARBALL and point that to either armnn-32.tar.xz or armnn.tar.xz... then add the other link as a comment.